stdlib/3/sys.pyi

```diff
@@ -5,120 +5,132 @@
 
 from typing import (
     List, NoReturn, Sequence, Any, Dict, Tuple, TextIO, overload, Optional,
-    Union, TypeVar, Callable, Type,
+    Union, TypeVar, Callable, Type
 )
 import sys
-from types import FrameType, TracebackType
+from types import FrameType, ModuleType, TracebackType
+
+if sys.version_info >= (3, 3):
+    from importlib.abc import MetaPathFinder
+else:
+    MetaPathFinder = Any
 
 _T = TypeVar('_T')
 
 # ----- sys variables -----
-abiflags = ...  # type: str
-argv = ...  # type: List[str]
+abiflags: str
+argv: List[str]
 base_exec_prefix: str
 base_prefix: str
-byteorder = ...  # type: str
-builtin_module_names = ...  # type: Sequence[str] # actually a tuple of strings
-copyright = ...  # type: str
+byteorder: str
+builtin_module_names: Sequence[str]  # actually a tuple of strings
+copyright: str
 # dllhandle = 0  # Windows only
 dont_write_bytecode: bool
-__displayhook__ = ...  # type: Any # contains the original value of displayhook
-__excepthook__ = ...  # type: Any  # contains the original value of excepthook
-exec_prefix = ...  # type: str
-executable = ...  # type: str
-float_repr_style = ...  # type: str
+__displayhook__: Any  # contains the original value of displayhook
+__excepthook__: Any  # contains the original value of excepthook
+exec_prefix: str
+executable: str
+float_repr_style: str
 hexversion: int
-last_type = ...  # type: Any
-last_value = ...  # type: Any
-last_traceback = ...  # type: Any
+last_type: Optional[Type[BaseException]]
```

Comment:
This change might cause new errors for code that uses `sys.last_*` blindly without checking for None. Probably not a big deal though.

Reply:
But these attrs *can* be `None`. I would say that previous stub was incorrect.